cloudflare/Cargo.toml

```diff
@@ -19,6 +19,9 @@ async-trait = "0.1.77"
 reqwest-middleware = "0.2.4"
 reqwest = { version = "0.11", default-features = false }
 async-std = "1.12.0"
+wasm-logger = "0.2.0"
+log = "0.4.20"
+walkdir = "2.4.0"
```

Comment:
where are we using this?

Reply:
walkdir is not needed anymore, I switched to pathbuf. wasm-logger and log is optional if we need logging at all